srv/pom.xml

```diff
@@ -104,6 +120,19 @@
 			<artifactId>spring-boot-devtools</artifactId>
 			<optional>true</optional>
 		</dependency>
+        <dependency>
+            <groupId>com.sap.cds</groupId>
+            <artifactId>cds-feature-xsuaa</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.sap.cloud.security.xsuaa</groupId>
+            <artifactId>xsuaa-spring-boot-starter</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.sap.cds</groupId>
+            <artifactId>cds-feature-hana</artifactId>
+        </dependency>
```

Comment:
Those dependencies automatically come with cds-starter-cloudfoundry. No need to add them here..

Reply:
Ok, I'll remove them and retry.